### ISSUE TITLE
Can't run Glances as a service in export mode? [windows]

### ISSUE THREAD

**Issue by seanvree** [nicolargo/glances#987]
Running Glances on Windows 10U as a service (NSSM)
Python: 3.5 64 Bit
Glances v2.7.1 with psutil v5.0.0 (running as a windows source via NSSM)

So, I'm using NSSM to run glances on my headless windows 10E server. 

Trying to get glances to run as a service while exporting data to influxdb.  I've got glances running as a service fine, the problem is that you have to use the -W switch to get the service to start. 

So, if I need to ALSO feed the data into influxdb, how would I do this since Glances won't run in -W mode AND export mode simultaneously?  

Just wondering if anyone else has tackled this. 

THANKS!

**Comment by quonic** [comment]
I have it running as a scheduled task.
Settings of task:

User: LOCAL SERVICE
Run whether user is logged on or not
Trigger: At log on
Action: Start a program
 - Program: C:\Python27\Scripts\glances.exe
 - Add arguments: -w
Conditions: I toggle everything off
Settings: What I have checked
 - Allow task to be run on demand
 - Stop the task if it runs longer than 3 days
 - In the running task does not end when requested, for it to stop
 - Do not start a new instance

I set the 3 days things as I normally don't have that computer running more than 24 hours at a time. The reason why I run it under the LOCAL SERVICE account is to prevent it from showing up as a command prompt in the task bar. I use [this](https://github.com/CiaccoDavide/BackgroundPage) little guy to get the local glance to show up as the desktop background.

I don't know if that helps, but there should be able to be a way to use powershell or something to setup a service or scheduled task. Something that Chef and what not can utilize to get it setup as a service.
